simulator-ui/tests/helpers/helper-functions.ts

```diff
@@ -0,0 +1,94 @@
+import { expect, Page } from '@playwright/test';
+import { navbarElementLinkPair } from './helper-interfaces';
+
+export const elementLinkPairNoDropdown: navbarElementLinkPair[] = [
+  { testName: 'navigationScenariosLink', link: /.*scenario*/, apiLink: '**/api/scenarios*' },
+  { testName: 'navigationScenarioExecutionsLink', link: /.*scenario-result*/, apiLink: '**/api/scenario-executions*' },
+];
+export const elementLinkPairEntityDroptdown: navbarElementLinkPair[] = [
+  { testName: 'navigationEntitiesMessageLink', link: /.*\/message*/, apiLink: '**/api/messages*' },
+  { testName: 'navigationEntitiesMessageHeaderLink', link: /.*\/message-header*/, apiLink: '**/api/message-headers*' },
+  {
+    testName: 'navigationEntitiesScenarioExecutionLink',
+    link: /.*\/scenario-execution*/,
+    apiLink: '**/api/scenario-executions*',
+  },
+  { testName: 'navigationEntitiesScenarioActionLink', link: /.*\/scenario-action*/, apiLink: '**/api/scenario-actions*' },
+  {
+    testName: 'navigationEntitiesScenarioParameterLink',
+    link: /.*\/scenario-parameter*/,
+    apiLink: '**/api/scenario-parameters*',
+  },
+  { testName: 'navigationEntitiesTestResultLink', link: /.*\/test-result*/, apiLink: '**/api/test-results*' },
+  { testName: 'navigationEntitiesParameterLink', link: /.*\/test-parameter*/, apiLink: '**/api/test-parameters*' },
+];
+// what to use as type instead of Promise<any>??
```

Comment:
as far as I see it does not return anything. `Promise<void>` would therefore be applicable. right?

Reply:
yes I think this should be correct and working.